### PR TITLE
GCN Circulars style guide addition

### PR DESCRIPTION
**Pull request by eburnsastro**
Added text for the GCN Circulars Style Guide to the lower half of the existing URL. Now with acknowledgements

Discussion for next TACH meeting:
-Are we removing the line length requirement? Are line breaks still an issue? If these are not issues we can remove them from the Style Guide

### CODE REVIEW

**Comment by lpsinger** [comment]
Would you please move the style guide to the documentation section and create a link to it?

**Comment by lpsinger** [comment]
Would you please move the style guide to the documentation section and create a link to it?

**Comment by jracusin** [comment]
Line breaks are still an issue with the way GCN circulars are processed.  While it's outdated that people still use plain text editors en mass, let's just keep this until we do an overhaul of the GCN circulars.  I'd like to solve the line break issue when we get to that.

File: app/routes/docs/styleguide.mdx [hunk 1]
```diff
@@ -0,0 +1,71 @@
+---
+meta:
+  title: GCN - Contributing
```
Comment (lpsinger) [inline app/routes/docs/styleguide.mdx:3]:
```suggestion
  title: GCN - Circulars Style Guide
```

File: app/routes/docs/styleguide.mdx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+---
+
+import {
+  ProcessList,
+  ProcessListItem,
+  ProcessListHeading,
+} from '@trussworks/react-uswds'
+
```
Comment (lpsinger) [inline app/routes/docs/styleguide.mdx:11]: Remove unused imports.

File: app/routes/circulars.mdx [hunk 3]
```diff
@@ -35,7 +35,7 @@ You can request access through the [sign up page](https://gcn.gsfc.nasa.gov/gcn_
 
 To submit a GCN Circular send an email to gcncirc@capella2.gsfc.nasa.gov. It is recommended that you copy [Scott Barthelmy](mailto:scott.d.barthelmy@nasa.gov).
 
-Please follow the [style guidelines](#style-guidelines) below. In particular, the email subject must contain at least one of the following case-sensitive strings:
+Please follow the [style guidelines](/docs/styleguide) below. In particular, the email subject must contain at least one of the following case-sensitive strings:
```
Comment (lpsinger) [inline app/routes/circulars.mdx:38]: It's no longer below.

**Review by lpsinger — CHANGES_REQUESTED** [review]
Please change the file extension from `.mdx` to `.md`. (You only need the `x` if you are including React components in your Markdown.)

File: app/routes/docs.tsx [hunk 4]
```diff
@@ -21,6 +21,9 @@ export default function Docs() {
             <NavLink key="client" to="client">
               Client Configuration
             </NavLink>,
+            <NavLink key="styleguide" to="styleguide">
+              Style Guide
```
Comment (lpsinger) [inline app/routes/docs.tsx:25]:
```suggestion
              Circulars Style Guide
```

File: app/routes/docs/styleguide.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
+---
+meta:
+  title: GCN - Circulars Style Guide
+---
+
+## Style Guide
+
+Every Circular begins with the header information described above, determined by the GCN system, the email subject line, and the approved submitted. The content within the circular should always begin with the ordered author list with affiliations. This is then followed by the scientific content.
```
Comment (lpsinger) [inline app/routes/docs/styleguide.md:8]: Above? Where?

File: app/routes/docs/styleguide.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
+---
+meta:
+  title: GCN - Circulars Style Guide
+---
+
+## Style Guide
```
Comment (lpsinger) [inline app/routes/docs/styleguide.md:6]: How about an example circular?

**Review by lpsinger — CHANGES_REQUESTED** [review]

File: app/routes/docs/styleguide.md (hunk 6 above)
Comment (eburnsastro) [reply app/routes/docs/styleguide.md:6]: Annoyingly the character spacing length of circulars is longer than that of the webpage inset functionality, so it has seemingly random line spacing. We can modify the circular text so it looks nice, or edit the page width

**Review by lpsinger — APPROVED** [review]

**Pull request merged**